Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/admission/AdmissionSharedViewModel.kt

```diff
@@ -0,0 +1,47 @@
+package de.rki.coronawarnapp.covidcertificate.person.ui.admission
+
+import androidx.annotation.VisibleForTesting
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import de.rki.coronawarnapp.ccl.dccadmission.model.DccAdmissionCheckScenarios
+import de.rki.coronawarnapp.tag
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import timber.log.Timber
+
+class AdmissionSharedViewModel(private val savedState: SavedStateHandle) : ViewModel() {
```

Comment:
Why do we actually need this shared ViewModel?

Reply:
Why can't we just pass `DccAdmissionCheckScenarios` to `AddmissionScenarios` after the calculation?

Reply:
I don't know how big it could be, there is a limit of how much data should be passed through bundle + you need to make it parcelable , I prefer to use VM in this case, does it cause any issues?

Reply:
Ah, makes sense. No, it doesn't cause any issues, I just want to figure out if there could be a simpler solution for this...

I think we have to wait for a result of the discussion that Chilja and Max are currently having and then continue to think about it.

Reply:
Let's leave it as is for now.